packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md

```diff
@@ -64,6 +64,7 @@ interact with the Dart Tooling Daemon from a DevTools
 extension. - [#7752](https://github.com/flutter/devtools/pull/7752)
 * Fix a DevTools routing bug related to disabling an
 extension. - [#7791](https://github.com/flutter/devtools/pull/7791)
+* Display query parameters for requests
```

Comment:
please put this in the section for the Network tool, and add your PR as a link (following the format that you see here for existing entries)